docs/getting-started/webstorm-pycharm.mdx

```diff
@@ -13,13 +17,100 @@ You'll want to get a free student license to avoid paying the subscription fee.
 
 Sign up for the Jetbrains Educational License [here](https://www.jetbrains.com/community/education/#students)
 
+Once you click on the link, you should be brought to this page. Under -> 'For Students and teachers', click 'Apply Now'
+
+[//]: # ([Jetbrains Apply Now]&#40;./img/jetbrains_license.png&#41;)
+
+Then fill in your details and accept the terms and conditions as follows:
```

Comment:
Without the images, "as follows" is out of context.

Reply:
Good catch, I have removed the unnecessary wording.